### PR TITLE
Add distribution management

### PR DESCRIPTION
**Pull request by andresrc**
@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -70,6 +70,18 @@
...
   </pluginRepositories>
 
+  <distributionManagement>
+    <repository>
+      <uniqueVersion>false</uniqueVersion>
+      <id>maven.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/releases/</url>
+    </repository>
+    <snapshotRepository>
+      <id>maven.jenkins-ci.org</id>
```
Comment (varyvol) [inline pom.xml]: Can the ids be the same?

Comment (andresrc) [inline pom.xml]: Yes, it is the same than in the plugin parent POM.

**Comment by varyvol** [comment]
🐝

**Comment by mafraba** [comment]
🐝

**Pull request merged**